### DISCUSSION TITLE
Success Alert

### DISCUSSION THREAD

**Discussion by mindytom** [twilio-labs/paste#2220] [Q&A] [answered]
Hi DSYS team,

I have a use case where after a user uploads a TLS certificate they can either get back a success, warning, or error validation. I'm thinking of using alerts for all three of the validations. Each time a user comes to the page the validation will happen so an alert will always be present to let them know the state of their TLS certificate upload is. I don't see a success alert component and wanted to check with the team to understand if there's guidance around this? 

Here's a [clickthrough prototype](https://www.figma.com/proto/QKL8U8ppyYS9l749sLGVrH/Link-Shortening?page-id=1110%3A151662&node-id=1110%3A151794&viewport=434%2C48%2C0.23&scaling=min-zoom&starting-point-node-id=1110%3A152108) of the different validations 

Screenshots of the success alert: 
<img width="628" alt="Success" src="https://user-images.githubusercontent.com/65037539/154373760-4369e5b9-34de-4fbf-916e-bf82b0c9a7c4.png">

**Comment by TheSisb** [comment, marked as answer]
Hi @mindytom,

We currently do not have a success variant for Alerts in Paste because they are designed for high-priority or time-sensitive [information that isn't triggered by a user-action](https://paste.twilio.design/components/alert#about-alerts).

What I've seen others do in similar scenarios is use a [Toast](https://paste.twilio.design/components/toast) for the immediate feedback and then display the status in a [Card](https://paste.twilio.design/components/card) or [Table](https://paste.twilio.design/components/table) using our [Status pattern](https://paste.twilio.design/patterns/status).

**Comment by gloriliale** [comment]
Hi @mindytom! [Here's a mockup](https://www.figma.com/file/SpeLpBoqN2vZv8KWcSn0J1/Paste-Feedback?node-id=1121%3A97355) of what using a status could look like for this page. Let us know what you think!

**Reply by mindytom** [reply]
Thanks Glorili! Using the status and toasts makes a lot of sense. Appreciate the mockup as it was very clear with the notes.
